components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/persistence/JDBCPersistenceManager.java

```diff
@@ -116,30 +116,38 @@ public void initializeDatabase() {
         dbInitializer.createIdentityDatabase();
     }
 
+    @Deprecated
```

Comment:
Need a comment why depreciated and the alternative

Reply:
Added